src/components/Search/SearchContext.tsx

```diff
@@ -125,7 +125,13 @@ function SearchContextProvider({children}: ChildrenProps) {
     const clearSelectedTransactions: SearchContextProps['clearSelectedTransactions'] = useCallback(
         (searchHashOrClearIDsFlag, shouldTurnOffSelectionMode = false) => {
             if (typeof searchHashOrClearIDsFlag === 'boolean') {
-                setSelectedTransactions([]);
```

Comment:
What is the limitation of passing an empty array to `setSelectedTransactions`? It looks like it's supposed to set `selectedTransactionIDs` automatically.
Can we modify `setSelectedTransactions` instead?